### ISSUE TITLE
2 Max Poolings in a row

### ISSUE THREAD

**Issue by DocDriven** [dandelionmane/tf-dev-summit-tensorboard-tutorial#5]
Hi,

thank you for the great tutorial. BUT: when running this with a single convLayer, you use Pooling twice. The first time it pools within the convLayer and the second time right after:

if use_two_conv:
    conv1 = conv_layer(x_image, 1, 32, "conv1")
    conv_out = conv_layer(conv1, 32, 64, "conv2")
  else:
    conv1 = conv_layer(x_image, 1, 64, "conv")
    **conv_out = tf.nn.max_pool(conv1, ksize=[1, 2, 2, 1], strides=[1, 2, 2, 1], padding="SAME")**

Is this just to keep the dimension of the flattened tensor the same?

**Comment by teamdandelion** [comment]
Thanks for the report. Fixed it.

**Comment by Marfly851** [comment]
With that change there is a problem of dimensionality between logits_size[400,10] and labels_size[100,10]. I'm not sure how to fix it.

**Comment by teamdandelion** [comment]
@Marfly851 sorry about that. I've fixed it.
